imagevector/images.yaml

```diff
@@ -119,22 +119,22 @@ images:
       availability_requirement: low
 - name: cluster-autoscaler
   sourceRepository: github.com/gardener/autoscaler
-  repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
+  repository: europe-docker.pkg.dev/gardener-project/releases/gardener/autoscaler/cluster-autoscaler
```

Comment:
@ccwienk Do the old images get transitioned to the new registry automatically, or does a release need to be made on the respective repo?
I am asking because `europe-docker.pkg.dev/gardener-project/releases/gardener/autoscaler/cluster-autoscaler:v1.27.1` is unavailable.

Reply:
@acumino : for the current state, I used a script to replicate all images (or so I hoped). In case I missed some images, those need to be replicated. I copied the autoscaler-image you mentioned (just now). Future versions will be released to Artifact-Registry, so this should be a transient issue.